src/time-slot-bar-week/time-slot-bar-week-days.tsx

```diff
@@ -0,0 +1,494 @@
+import dayjs, { Dayjs } from "dayjs";
+import { useMemo, useRef, useState } from "react";
+import { InternalCalendarProps } from "../shared/internal-calendar";
+import {
+    GrowDayCell,
+    InteractiveCircle,
+    StyleProps,
+} from "../shared/internal-calendar/standard";
+import { Text } from "../text/text";
+import { TimeSlot as TimeSlotComponent } from "../time-slot-bar/time-slot-bar.styles";
+import { TimeSlot } from "../time-slot-bar/types";
+import { CalendarHelper } from "../util/calendar-helper";
+import {
+    CollapseExpandAllButton,
+    CollapseExpandAllWrapper,
+    ColumnWeekCell,
+    DayLabelWeek,
+    HeaderCellWeek,
+    HeaderCellWeekColumn,
+    TimeColumn,
+    TimeColumnText,
+    TimeColumnWrapper,
+    TimeSlotWrapper,
+    Wrapper,
+} from "./time-slot-bar-week-days.style";
+import {
+    flatMapDeep,
+    isEmpty,
+    last,
+    mapValues,
+    maxBy,
+    minBy,
+    reduce,
+    times,
+    values,
+} from "lodash";
```

Comment:
can look through to see which functions have a ES6 javascript equivalent to avoid including too many lodash dependencies. mapValues can be replaced by `Object.values` and reduce can be replaced by `[].reduce()`

for others, use the sub imports instead i.e. `import isEmpty from "lodash/isEmpty";`